### PR TITLE
Add jpeg parser

### PR DESCRIPTION
**Pull request by s4chin**
Updated the kaitaistruct version here

### CODE REVIEW

**Comment by Kriechi** [comment]
I don't see any jpeg-related changes in this PR...?

**Comment by s4chin** [comment]
There aren't any yet. I'm working on this :)

File: mitmproxy/contrib/kaitaistruct/exif.py [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+
+import array
+import struct
+import zlib
+from enum import Enum
+
+from kaitaistruct import KaitaiStruct, KaitaiStream, BytesIO
+
+
+class Exif(KaitaiStruct):
```
Comment (s4chin) [inline mitmproxy/contrib/kaitaistruct/exif.py]: The only variables I get from the `app1` segment are the ones in the `__init__`. The `exif` data is never parsed.

**Comment by s4chin** [comment]
The test fails because the `app13` segment hasn't been specified in the declaration. I'll write the complete declaration later and will try to merge it in `kaitaistruct` before using it here. But my problem with the `app1` segment remains.

**Comment by Kriechi** [comment]
@s4chin looks good! Unfortunately this block of code is not covered any more with tests:
https://github.com/s4chin/mitmproxy/blob/8eb1d34644c454a79c168f6d68fc0ae45bbc4d1a/mitmproxy/contentviews/image/view.py#L55-L62
If it is still needed, please add tests. Otherwise just delete the whole thing.

**Comment by s4chin** [comment]
Done!

File: test/mitmproxy/contentviews/test_image_parser.py [hunk 2]
```diff
@@ -104,3 +104,50 @@ def test_parse_png(filename, metadata):
...
+        ('jfif_version', '(1, 1)'),
+        ('jfif_density', '(72, 72)'),
+        ('jfif_unit', '1'),
+        ('make', 'Canon'),
+        ('model', 'Canon PowerShot A60'),
+        ('modify_date', '2004:07:16 18:46:04'),
+        ('Size', '717 x 558 px')
+    ],
+    # check multiple segments
+    "mitmproxy/data/image_parser/image-err1.jpg": [
```
Comment (mhils) [inline test/mitmproxy/contentviews/test_image_parser.py]: This picture is copyrighted (http://www.gettyimages.de/license/93901920), do we have another one which could replace this test case?

Comment (s4chin) [inline test/mitmproxy/contentviews/test_image_parser.py]: I just used the image which was being used in `test_image.py`

**Pull request merged**